packages/main/src/themes/TabInStrip.css

```diff
@@ -258,7 +258,7 @@
 	padding: 0 0.188rem;
 }
 
-.ui5-tab-strip-item--textOnly.ui5-tab-strip-item--withAddionalText {
```

Comment:
There is one more place where `withAddionalText` is mispelled in the code. Could you fix it there too?